### PR TITLE
ER-1242 new applications scheduled notifications

### PR DESCRIPTION
**Pull request by sachpatel**

### CODE REVIEW

File: src/Jobs/Recruit.Vacancies.Jobs/Communication/WeeklyApplicationsSubmittedTimeTrigger.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            _timeProvider = timeProvider;
+        }
+
+        public Task WeeklyApplicationsSubmittedCreateAggregateRequestAsync([TimerTrigger(Schedules.EightAmWeeklySunday)] TimerInfo timerInfo, TextWriter log)
+        {
+            const int eightAmHour = 8;
+            _logger.LogInformation($"Timer trigger {this.GetType().Name} fired");
+
+            var from = _timeProvider.Today.AddDays(-7).AddHours(eightAmHour);
+            var to = _timeProvider.Today.AddHours(eightAmHour);
```
Comment (chrisjensenuk) [inline src/Jobs/Recruit.Vacancies.Jobs/Communication/WeeklyApplicationsSubmittedTimeTrigger.cs]: Maybe do this
```
var to = _timeProvider.Today.AddHours(eightAmHour);
var from = to.AddDays(-7);
```

Comment (sachpatel) [reply src/Jobs/Recruit.Vacancies.Jobs/Communication/WeeklyApplicationsSubmittedTimeTrigger.cs]: 👍

**Comment by sachpatel** [comment]
Can do a demo, will setup some data and then show who ever is interested after standup

File: src/Jobs/Recruit.Vacancies.Jobs/Communication/DailyApplicationsSubmittedTimeTrigger.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            _timeProvider = timeProvider;
+        }
+
+        public Task DailyApplicationsSubmittedCreateAggregateRequestAsync([TimerTrigger(Schedules.EightAmDaily)] TimerInfo timerInfo, TextWriter log)
+        {
+            const int eightAmHour = 8;
+            _logger.LogInformation($"Timer trigger {this.GetType().Name} fired");
+
+            var from = _timeProvider.Today.AddDays(-1).AddHours(eightAmHour);
+            var to = _timeProvider.Today.AddHours(eightAmHour);
```
Comment (chrisjensenuk) [inline src/Jobs/Recruit.Vacancies.Jobs/Communication/DailyApplicationsSubmittedTimeTrigger.cs]: Same here.
```
var to = _timeProvider.Today.AddHours(eightAmHour);
var from = to.AddDays(-7);
```

**Pull request merged**